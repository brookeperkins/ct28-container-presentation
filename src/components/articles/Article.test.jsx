import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Article from './Article';

describe('Article component', () => {
  afterEach(() => cleanup());
  it('renders Article', () => {
    const { asFragment } = render(<Article
      title="Check out this TITLE"
      author="Ida B. Wells"
      description="This is a description of some sort."
    />);
    expect(asFragment()).toMatchSnapshot();
  });
});
