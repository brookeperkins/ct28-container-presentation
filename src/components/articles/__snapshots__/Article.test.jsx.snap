// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Article component renders Article 1`] = `
<DocumentFragment>
  <h3>
    Check out this TITLE
  </h3>
  <p>
    By: Ida B. Wells
  </p>
  <p>
    Summary: This is a description of some sort.
  </p>
</DocumentFragment>
`;
